tests/integration-tests/tests/create/test_create/test_create_wrong_os/pcluster.config.ini

```diff
@@ -1,21 +0,0 @@
-[global]
-cluster_template = default
-
-[aws]
-aws_region_name = {{ region }}
-
-[cluster default]
-base_os = {{ os }}
-key_name = {{ key_name }}
-vpc_settings = parallelcluster-vpc
-scheduler =  {{ scheduler }}
-master_instance_type = {{ instance }}
-custom_ami = {{ custom_ami }}
-master_root_volume_size = 100
```

Comment:
missing volume size and use public ips options

Reply:
Default volume size was increased to 35, so the specification is no longer needed.
A False default of ElasticIp will be added to the config layer.
So neither of the parameter is necessary now